pkg/api/gen/priv/secrets/v1alpha1/secrets.pb.go

```diff
@@ -20,7 +20,7 @@ const (
 	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
 )
 
-// Secret is a resource that represents a Jetify Secret inside a Project.
+// Secret is a resource that represents a Jetpack Secret inside a Project.
```

Comment:
undo

Reply:
If this is auto-gen it probably means a comment in the .proto file needs to be updated?